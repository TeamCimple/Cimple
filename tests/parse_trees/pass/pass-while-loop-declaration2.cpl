int main() {
	while(1) {
		while(2) {
			
		}
	}
}
