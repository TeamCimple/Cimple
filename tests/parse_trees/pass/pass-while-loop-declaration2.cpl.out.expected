FuncDecl(Name: Var(main) ReturnType: int Parameters:  Body: CompoundStatement(Declarations:  StatementList: WHILE Int(1) CompoundStatement(Declarations:  StatementList: WHILE Int(2) CompoundStatement(Declarations:  StatementList: )))) 
